src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -39,6 +40,40 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("Exceptions", LoaderExceptions, typeof(Exception[]));
         }
 
+        public override string Message
+        {
+            get
+            {
+                if (LoaderExceptions.Length == 0)
+                {
+                    return base.Message;
+                }
+
+                StringBuilder text = new StringBuilder();
+                text.AppendLine(base.Message);
+                foreach (Exception e in LoaderExceptions)
+                {
+                    text.Append('(');
+                    text.Append(e.Message);
+                    text.AppendLine(") ");
```

Comment:
AE keeps everything on one line. I am not sure whether we try to keep exception messages on a single line. Up to you but if you keep multiple lines I don't think you need parentheses.